src/auth/login/login.scss

```diff
@@ -57,3 +57,32 @@
     }
   }
 }
+
+.login-page {
+  .user-details {
+    .user-data {
+      p {
+        font-family: Inter;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 12px;  
+        line-height: 15px;
+      }
+      a {
+        button {
+          background: #1A73E8;
+          box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.08);
+          border-radius: 5px;
+          .loginbtn-content {
+            font-family: Inter;
```

Comment:
This is not a good way to use fonts

Reply:
@rak-shit in scss you have `scss variable` please make use of that.
[1]. https://sass-lang.com/documentation/variables

Reply:
Okay. I will see that.

Reply:
> This is not a good way to use fonts

Hey @vaibhavdaren, can you please elaborate a bit. I mean are you saying about the usage of fonts for each element separately or the way I am using it for each element.

Reply:
I have made the changes.